src/test/resources/application-test-a58-transform.yaml

```diff
@@ -5,7 +5,7 @@ uk:
         caseTypeofApplication:
           A58_STEP_PARENT:
             caseFields:
-              CASE_TYPE_ID: A58
```

Comment:
Other day in one of the excel sheet @yogendraU  was sharing, it was A58 for all adoption case types, if there is update in this value as ADOPTION then it's OK

Reply:
yeah we will change it later point of time. now we keep ADOPTION as case type